packages/components/src/utils/log/warnIfDeprecatedPropValueIsUsed.spec.ts

```diff
@@ -0,0 +1,57 @@
+import * as loggerUtils from "./logger";
+import * as helperUtils from './helper';
+import { warnIfDeprecatedPropValueIsUsed } from "./warnIfDeprecatedPropValueIsUsed";
+
+const warningMessage1 = "prop='deprecatedValue' is deprecated for component span and will be removed with next major release.";
+const warningMessage2 = "Please use prop='value' instead.";
+
+class SomeInstance {
+  host = document.createElement('span');
+  prop: string | boolean | number = undefined;
+}
+
+const SOME_INSTANCE_PROP_DEPRECATED = ['deprecatedValue'] as const;
+type SomeInstancePropDeprecated = typeof SOME_INSTANCE_PROP_DEPRECATED[number];
+const SOME_INSTANCE_PROP = [
+  'value',
+  ...SOME_INSTANCE_PROP_DEPRECATED,
+] as const;
+type SomeInstanceProp = typeof SOME_INSTANCE_PROP[number];
+
+it('should call getDeprecatedPropOrSlotWarningMessage() util with correct parameters when prop value is key of deprecationMap', () => {
```

Comment:
Tests suggest that calling `getDeprecatedPropOrSlotWarningMessage` does not really do anything.